### ISSUE TITLE
Update requirements.txt

### ISSUE THREAD

**Issue by dziban303** [sniffingpickles/Auto-Amazon-Giveaways#2]
I submitted a pull request to update requirements.txt, adding bs4 and tinydb.

**Comment by distrax** [comment]
I think you've covered the new issues well. The requirements.txt update led my script to run but I can't get more than two done at the moment due to the other errors. <br><br>This is in no way complaining. Just confirming. Great work sniffingpickles and good find dziban303

**Comment by dziban303** [comment]
@distrax while this gets fixed, in the meantime you can use [my* fork](https://github.com/dziban303/Auto-Amazon-Giveaways) of zdrouse's original bot.

It works pretty well, but it doesn't do video giveaways. Run it with `python loop.py give_it_away_now.py` to have it restart when it crashes, and add your username and password in give_it_away_now.py so you won't need to enter it every time it starts. It will open a random giveaway page so if it does crash, it won't waste time trying all the giveaways you've already done.

You may need to add the full path to your python.exe in loop.py, if you've got multiple python versions installed, or don't have the directory in your PATH variable.

* Basically all the changes from zdrouse's original was done by sniffingpickles, I just combined it all in one.

**Comment by sniffingpickles** [comment]
Thanks for the updates/pull requests, I've been busy irl; so I just dumped the giveaway.py changes without thinking of the import/libs change. 

Let me know if i can update anything else, I'm going to update requirements today when I get the chance.  Sorry about that.

Video giveaways should be fine, there's one new giveaway for a kindle that's messing everything up; I'll try and get a working query finder for it tonight.
